src/filters/filter_cue.cpp

```diff
@@ -60,19 +60,22 @@ namespace dmxfish::filters {
         }
         try {
             switch (channel.at(nr_channel).channel_type) {
-                case EIGHT_BIT:
+                case EIGHT_BIT: {
                     cues.at(cue).eight_bit_frames.push_back(
-                            key_frame<uint8_t>(std::stoi(str.substr(start, sep - start)), tr));
+                            key_frame<uint8_t>((uint8_t) std::stoi(str.substr(start, sep - start)), tr));
```

Comment:
I would suggest using `min` and `max` here to create meaningful boundaries as the simple cast will simply cut away the excess bits and this might cause bad behavior in case of small offsets. We have the time to do this during parsing as `stoi` is slow anyway.

Reply:
Changed within commit 0ada92d